pkg/cluster/hash_test.go

```diff
@@ -35,38 +35,35 @@ func TestHashScaleSetStability(t *testing.T) {
 		"v3.2": {
 			{
 				role: api.AgentPoolProfileRoleMaster,
-				// jminter: e436... value verified manually against v3.2
 				// this value should not change
 				expectedHash: "e43619d6476b958470ec8e95f83c1023f15c63bb7d896adc2246cc1d9da68d0e",
 			},
 			{
 				role: api.AgentPoolProfileRoleInfra,
-				// jminter: d438... value verified manually against v3.2
 				// this value should not change
 				expectedHash: "d438d7076d6a3495d0663323dd81600b389f992499750df0a859d208d9fa42f7",
 			},
 			{
 				role: api.AgentPoolProfileRoleCompute,
-				// jminter: fc64... value verified manually against v3.2
 				// this value should not change
 				expectedHash: "fc64527a7a3fc02568083f200de97c9fcb22aea448e57ae46e58f1a76ba1f6f6",
 			},
 		},
-		"v4.0": {
+		"v4.1": {
 			{
 				role: api.AgentPoolProfileRoleMaster,
 				// this value should not change
-				expectedHash: "49de607e3b50aff9f2fa1a64e7a6173ab2c13969dcdc9036623ddc0666ca99af",
+				expectedHash: "9a4b049bda1bda0bf394b6e5a21c89bb8b219346261de0e6b01d2efb100f3e01",
```

Comment:
Is how to generate and verify these hashes documented somewhere?

Reply:
run the test, let it fail and output what it expects to see, copy paste from there
*cough*